src/main/java/uk/gov/hmcts/darts/arm/util/ArmResponseFilesUtil.java

```diff
@@ -37,4 +41,28 @@ public static String getObjectTypeId(ExternalObjectDirectoryEntity externalObjec
         }
         return objectTypeId;
     }
+
+    public static Integer getRelationIdFromArmResponseFilesInputField(String input) {
+        try {
+            String relationId = findFirstPatternMatch("\"relation_id\".*?:.*?\"([^\"]+)\"", input);
+            return Integer.parseInt(relationId);
+        } catch (Exception e) {
+            log.error("Error: {}", e.getMessage());
+        }
+        return null;
+    }
+
+    public static String getOperationFromArmResponseFilesInputField(String input) {
+        return findFirstPatternMatch("\"operation_id\".*?:.*?\"([^\"]+)\"", input);
```

Comment:
Dependent on the response from the previous comment it maybe better to assume we always get valid json return then grab the operation_id from this valid json.

If the response from previous comment is that we still want to grab these details from invalid JSON then this comment can be ignored :)

Reply:
See Hemanta's comments

Reply:
this has been removed